### ISSUE TITLE
Active sub-menu entries also don’t meet contrast requirements

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/text#2829]
Active menu entries also don’t meet contrast requirements, here possibly also work with a dot marker, or left border?

Example would be the different heading styles in the menubar.

May need upstream work in the component library if we reuse the active state from there.

**Comment by vinicius73** [comment]
To fix it, we must change the `NcActions` component

https://github.com/nextcloud/nextcloud-vue/blob/e28a7178ca938e7175c1c78ee29055f59cdffaef/src/components/NcActions/NcActions.vue#L1037-L1040

## Actual

![image](https://user-images.githubusercontent.com/1561347/186540697-c21f5b9c-9ab6-4a96-ae4c-461aced090d8.png)

## Expected 

### Option 1
![image](https://user-images.githubusercontent.com/1561347/186540795-2add7665-023f-4b6c-9b4d-4c6ea7d997cd.png)

Just change the background of `.action-item--open .action-item__menutoggle` 

```css
background-color: var(--color-background-hover);
```

### Option 2
![image](https://user-images.githubusercontent.com/1561347/186541344-4fbd2c7a-3023-44e9-86c5-2356ba6af027.png)

Change the background of `.action-item--open .action-item__menutoggle` and add box shadow:

```css
background-color: var(--color-background-hover);
box-shadow: 0px 0px 0.2rem 0px var(--color-primary-light-text);
```

**Comment by juliushaertl** [comment]
@jancborchardt I think the blue bar that we already have should be enough, right? I'm wondering why those were not there during the review call.

**Comment by jancborchardt** [comment]
@juliushaertl yes, I’d say so too. However, this is not the new menu as per Vue-components? (With the padding and rounded corners)

@vinicius73 regarding the icon highlight: Option 1 looks good, but for contrast reasons we need to color the icon black instead of primary.
And small note: "Show Outline" → "Show outline", we only capitalize the first word as per design guidelines. :)

**Comment by vinicius73** [comment]
@jancborchardt I've adjusted the style to get closer to the current actions component.

![image](https://user-images.githubusercontent.com/1561347/187992595-366aa332-13e7-4213-ab10-37a59bae570b.png)

![image](https://user-images.githubusercontent.com/1561347/187992667-97e40d3d-3e37-4b57-8e5b-325a6cbecaf5.png)
